emissionsapi/openapi.yml

```diff
@@ -19,9 +19,16 @@ paths:
       operationId: emissionsapi.web.get_data
       description: |
           Get all points in [GeoJSON](https://geojson.org/) format.
+
+          You can provide multiple parameters. If you provide geoframe and country and polygon, only one will be used.<br>
+          The priority of the parameter is:<br>
+            1. geoframe
```

Comment:
Will this be properly rendered?

Reply:
This works fine on my windows setup. I did not find any other methode to chreate a linefeed here...

Reply:
We should test the rendering on a linux maschine!

Reply:
I would try to avoid inline HTML or forced formatting where possible. I you really need a newline here what about

```
You can provide multiple parameters. If you provide geoframe and country and polygon, only one will be used.\
The priority of the parameter is:
  1. geoframe
```

Reply:
This is what it looks like for the record

![image](https://user-images.githubusercontent.com/5557272/68158293-a4915d00-ff4f-11e9-8ff0-f74489aece57.png)

Reply:
So, we can just use Markdown here? Then let's use:

```
Get all points in [GeoJSON](https://geojson.org/) format.

You can provide multiple parameters.
If you provide geoframe and country and polygon, only one will be used.

The priority of the parameter is:

1. geoframe
2. country
3. …
```

Which should render like this:

> Get all points in [GeoJSON](https://geojson.org/) format.
>
> You can provide multiple parameters.
> If you provide geoframe and country and polygon, only one will be used.
>
> The priority of the parameter is:
>
> 1. geoframe
> 2. country
> 3. …

Reply:
Thanks, its nice to know how to enforce a linefeed in a less hacky way. But I will remove the <br> linefeeds since they are not really needed.